android/app-newm/src/main/java/io/newm/screens/library/screens/EmptyWalletScreen.kt

```diff
@@ -0,0 +1,54 @@
+package io.newm.screens.library.screens
+
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import io.newm.core.ui.buttons.PrimaryButton
+
+private const val RECORD_STORE_URL = "https://recordstore.newm.io/"
+
+@Composable
+fun EmptyWalletScreen() {
+    val context = LocalContext.current
+
+    Box(
+        contentAlignment = Alignment.Center,
+        modifier = Modifier.fillMaxSize()
+    ) {
+        Column(horizontalAlignment = Alignment.CenterHorizontally) {
+            Text(
+                text = "No NFTs yet?",
+                style = MaterialTheme.typography.h6,
+                textAlign = TextAlign.Center,
+                modifier = Modifier.padding(16.dp)
+            )
+            Spacer(modifier = Modifier.height(8.dp))
+            Text(
+                text = "You can buy NFTs from our marketplace",
```

Comment:
I'll need to add this to iOS too.  Added jira ticket.

Reply:
This is a placeholder, waiting to hear back from design about what the screen should look like and say.